packages/edit-navigation/src/components/menu-editor/style.scss

```diff
@@ -49,12 +49,40 @@
 	}
 }
 
+.block-editor-block-navigation-leaf {
+	transition: background 0.1s ease-out;
+	@include reduce-motion("transition");
+
+	> :first-child {
+		padding-left: $grid-unit-20;
+	}
+	> :last-child {
+		padding-right: $grid-unit-20;
+	}
+
+	&:hover {
+		background: $light-gray-300;
+	}
+
+	.block-editor-block-navigation-block-contents {
+		padding-top: $grid-unit-15;
+		padding-bottom: $grid-unit-15;
+	}
+}
+
```

Comment:
These changes should live in `components/block-navigation/style.scss` if they should also apply to the post editor block navigator.